src/components/ebay-combobox/component.js

```diff
@@ -115,14 +115,21 @@ module.exports = {
             this.focus();
         }
 
-        if (this.expander && this.expander.isExpanded() && !wasClickedOption && !this.buttonClicked) {
+        if (
```

Comment:
this file just had some leftover prettier changes... I can drop those from the PR if needed

Reply:
Yeah, let's drop any unrelated files.